### DISCUSSION TITLE
Twitter screenshot is always "Something went wrong" in Ubuntu.

### DISCUSSION THREAD

**Discussion by Abhi0725** [spatie/browsershot#787] [Bugs]
Hi 👋 

I tried this in my Intel Mac and it works fine for the following command:

`\Spatie\Browsershot\Browsershot::url('https://twitter.com/gauravmakhecha/status/1717201152973144129?s=42')->userAgent('Mozilla/5.0 (iPhone; CPU iPhone OS 14_7_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) FxiOS/36.0  Mobile/15E148 Safari/605.1.15')->delay(3000)->touch()->mobile()->windowSize(375, 812)->noSandbox()->save('public/test.png');`

The screenshot is correctly shown here:

<img width="384" alt="image" src="https://github.com/spatie/browsershot/assets/8065382/42ce76ac-7d8b-4a38-a2a2-eecca6caea46">

On the other hand, on Ubuntu 22.04.3 with the same above command I am getting this:

<img width="376" alt="image" src="https://github.com/spatie/browsershot/assets/8065382/2021928a-5c8a-446a-930b-98034e55c35f">

Is there something due to environment issue? On Ubuntu I installed puppeteer globally via npm.

**Comment by Abhi0725** [comment]
It seems that the issue is because Twitter blacklists AWS IPs? The error one is from EC2 instance.

**Reply by cas-technology** [reply]
_very_ possible that this is on the Twitter end or some rudimentary OS detection. I get the same error on a basic linux machine all the time if I'm not signed in. If I'm signed in, no issues.
